packages/@aws-cdk/cloudformation-include/lib/file-utils.ts

```diff
@@ -1,6 +1,68 @@
 import * as fs from 'fs';
+import * as yaml from 'yaml';
+import * as yaml_cst from 'yaml/parse-cst';
+import * as yaml_types from 'yaml/types';
+
+function makeTagForCfnIntrinsic(
```

Comment:
Is there any way we can move this function, and `shortForms`, below the `readJsonSync` and `readYamlSync` functions?

Files should generally read top-to-bottom, with the most generic, public members first, and then their implementation details in private members later.